client/deposits/index.tsx

```diff
@@ -100,7 +101,9 @@ const NextDepositNotice: React.FC = () => {
 
 const DepositFailureNotice: React.FC = () => {
 	const { hasErroredExternalAccount } = useAccountStatus();
-	const accountLink = wcpaySettings.accountStatus.accountLink;
+	const accountLink = addQueryArgs( wcpaySettings.accountStatus.accountLink, {
+		source: 'deposits',
```

Comment:
Could we be more specific about the source here? Maybe something like `deposit__failure-notice`.

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4